cekit/templates/template.jinja

```diff
@@ -46,24 +46,31 @@ EXPOSE {%- for port in helper.ports(ports) %} {{ port }}{% endfor %}
 USER root
 
 {% if packages and packages.install %}
-{% if packages.repositories %}
+{% if packages.repositories_injected %}
 # Add custom repo files
 COPY \
-{% for repo in packages.repositories %}
+{% for repo in packages.repositories_injected %} 
     repos/{{ repo.filename }} \
 {% endfor %}
     /etc/yum.repos.d/
 
 {% endif %}
+{% if packages.repositories %}
+{% for repo in packages.repositories %}
+{% if repo.present and repo.rpm %}
+RUN yum install -y {{ repo.rpm }} && yum clean all
```

Comment:
Why you are not doing the same thing here as for any other installed RPMs? `rm -rf /var/cache/yum && rpm -q RPM`